src/styles/tailwind.css

```diff
@@ -75,4 +59,66 @@ html {
   h6 {
     @apply mb-4;
   }
+
+  .border-outset {
+    border-style: outset;
+  }
+  .skip-main {
+    @apply absolute w-px h-px -left-96;
+  }
+
+  .skip-main:focus-within {
+    @apply absolute w-screen h-auto top-16 left-0 z-50 flex justify-center;
+  }
+
+  .layout-container {
```

Comment:
I had removed all of these layout-container classes, can we remove them since they are no longer being used? Believe lines 62-123 were all removed unless you had added any of them.

Reply:
Removed unused styles in tailwind.css and screen breakpoints from config